p2p/tests/peer_handshake.rs

```diff
@@ -76,6 +76,8 @@ fn peer_handshake() {
 		net_adapter,
 	).unwrap();
 
+	assert!(peer.info.user_agent.ends_with(env!("CARGO_PKG_VERSION")));
```

Comment:
We definitely don't want to do this.. this would exclude all peers with even a slightly different version than ours.. peers are generally okay to have different version numbers from our so long as their fork is compatible and they're not misbehaving.

Reply:
@yeastplume this in an integration test, it just checks that `user_agent` has been properly generated in our code. did I miss smth?

Reply:
No.. apologies that was a brainfart, all good.